.github/workflows/CI.yml

```
@@ -274,3 +274,21 @@ jobs:
         run: |
           SET CHERE_INVOKING=1
           C:\cygwin64\bin\bash -l -c "bash dev/ci.sh"
+
+  slack-notification-on-failure:
+    name: Send slack notification on CI failure
+    needs:
+      - test-unix
+      - test-cygwin
+    if: ${{ failure() && github.event_name == 'push' }}
```

Would something like be sensible (and would it work)?
```suggestion
    if: ${{ failure() && github.event_name == 'push' && github.repository == 'gap-system/gap' }}
```

---

I think that makes sense, yes. I'll have to add that once I'm done testing.